### DISCUSSION TITLE
iOptron HAZ31 mount support

### DISCUSSION THREAD

**Discussion by WimDeMeester** [indilib/indi#2065] [Q&A] [answered]
Hi all,

Is anyone able to connect to the iOptron HAZ31 mount?  According to the website of iOptron, an INDI driver is available (see [website](https://www.ioptron.com/product-p/hz312.htm)), but we are not able to connect to the mount.
We tried with the [INDI iOptronV3](https://indilib.org/devices/telescopes/ioptron/cem120.html) driver.

**Comment by knro** [comment]
what's the issue exactly? maybe just a baud rate issue?

**Reply by WimDeMeester** [reply]
We get a Connection Timed out error.  We tried with all available baud rates and this does not change anything...

**Reply by knro** [reply]
Please enable mount logging and attach log

**Reply by Wannes-VanPuyvelde** [reply]
Hi. Here are the screenshots of the enabled logging and the log itself: 
![image](https://github.com/indilib/indi/assets/111141230/9c2e8070-517b-4e96-a1fd-f409716256de)
![image](https://github.com/indilib/indi/assets/111141230/89858e5e-2fbc-4486-89d1-cf98bf0e764f)

**Reply by knro** [reply]
That's not it. Check the attached screenshot and then go to Ekos --> Logs --> Browse Logs and zip & attach the KStars log file. 115200 baud rate also not response?
![Screenshot_20240529_173537](https://github.com/indilib/indi/assets/5986347/61b0394d-8e7b-46f4-96b2-9381396c69cb)

**Comment by Wannes-VanPuyvelde** [comment, marked as answer]
We were using the wrong cable. It now works with an USB-B cable in the handcontroller
